app/src/main/java/org/apache/fineract/data/models/loan/LoanAccount.java

```diff
@@ -127,7 +126,7 @@ public void setLastModifiedBy(String lastModifiedBy) {
     }
 
     public LoanParameters getLoanParameters() {
-        return new Gson().fromJson(parameters, LoanParameters.class);
+        return loanParameters;
```

Comment:
I think I made this because of some code restriction, change it as it was when you will work in this then we will see this.